test/resources/microshift-host.resource

```diff
@@ -25,3 +25,15 @@ Login MicroShift Host
 Logout MicroShift Host
     [Documentation]  Close the open ssh connection to the MicroShift host
     SSHLibrary.Close Connection
+
+Reboot MicroShift Host
```

Comment:
This would be more robust if we looked for a different boot id after the reboot, to avoid any race condition with issuing the reboot command and sshd stopping. It doesn't seem like the most likely race, so let's make that change in another PR.